examples/quickstart/jupyter-notebooks/api-tutorial.ipynb

```diff
@@ -131,15 +177,27 @@
    },
    "outputs": [],
    "source": [
-    "endpoint = \"/status\"\n",
-    "print(f\"\\033[1mQuery endpoint\\033[0m: {druid_host+endpoint}\")\n",
-    "http_method = \"GET\"\n",
-    "\n",
-    "payload = {}\n",
-    "headers = {}\n",
-    "\n",
-    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
-    "print(\"\\033[1mResponse\\033[0m: : \\n\" + json.dumps(response.json(), indent=4))"
+    "response = session.get(endpoint)\n",
+    "json = response.json()\n",
+    "json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de82029e",
+   "metadata": {},
+   "source": [
+    "Since Druid's responses are JSON, and Requests converted the JSON to a set of Python dicts (maps) and arrays, we can easily pull out the information we want. For example, to seee just the version:"
```

Comment:
```suggestion
    "Since Druid's responses are JSON, and Requests converted the JSON to a set of Python dictionaries (maps) and arrays, we can easily pull out the information we want. For example, to see just the version:"
```

Reply:
"you" instead of "we"